client/driver/executor/executor.go

```diff
@@ -781,11 +789,13 @@ func (e *UniversalExecutor) aggregatedResourceUsage(pidStats map[string]*cstruct
 		totalSwap += pidStat.MemoryStats.Swap
 	}
 
+	totalTicks := e.systemCpuStats.TicksConsumed(percent)
 	totalCPU := &cstructs.CpuStats{
 		SystemMode: systemModeCPU,
 		UserMode:   userModeCPU,
 		Percent:    percent,
 		Measured:   ExecutorBasicMeasuredCpuStats,
+		TotalTicks: totalTicks,
```

Comment:
Just put `e.systemCpuStats.TicksConsumed(percent)` right here. Not used elsewhere